Sourcery/Models/Class.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+// sourcery: skipDescription, skipEquatable
+class Class: Type {
+    override var kind: String { return "class" }
+
+    /// contains all types inheriting from known BaseClass
+    // sourcery: skipEquality
+    var inherits = [String: String]()
```

Comment:
as much as I like having `Class` I've been thinking about this and I think we should include this properties in the base `Type` and just be empty or everyone will have to duplicate their templates because accessing supertype, inherits would crash the KVC templating engine. 

Usually template for classes / structs differs minimal, so have the `Class` type since it's useful but put those properties back in Type